ksql-rest-app/src/main/java/io/confluent/ksql/rest/client/properties/LocalProperties.java

```diff
@@ -44,9 +44,20 @@ public LocalProperties(final Map<String, Object> initial) {
    * @return the previous value for the property, or {@code null}.
    */
   public Object set(final String property, final Object value) {
+    return set(property, value, true);
+  }
+
+  /**
+   * Set property.
+   *
+   * @param property the name of the property
+   * @param value the value for the property
+   * @return the previous value for the property, or {@code null}.
```

Comment:
Need to document the 'useParsed` argument.

Reply:
Added docs for the new argument.